src/handlers/completion.rs

```diff
@@ -197,6 +197,198 @@ fn new_string_arg_completion(value: String) -> CompletionItem {
     }
 }
 
+fn new_param_completion(
+    name: String,
+    trigger: String,
+) -> CompletionItem {
+    let insert_text = if trigger == "(" {
+        format!("{}: ", name)
+    } else {
+        format!(" {}: ", name)
```

Comment:
ah so this is the if it is first arg for a func, then add no space, else add a space. Is it safe to assume this `new_param_completion` is only intended for funcs?

Reply:
right now yes, that might change in the future